### DISCUSSION TITLE
Adjust the way SV/volume points are displayed in osu!taiko editor timeline

### DISCUSSION THREAD

**Discussion by Hiviexd** [ppy/osu#20089] [Ideas]
currently, only sampleset/volumes are displayed on objects in the timeline, which is causing 2 issues:

- those stamps are pretty much useless compared to a much more important piece of a information which is  slider velocity, it would be much better and more useful to show SVs for objects in the timeline.
SV values currently only show on sliders which are insanely uncommon compared to notes.

![image](https://user-images.githubusercontent.com/62819481/188055844-4ccdf690-e98f-4cbe-8e4d-64d34a6665c3.png)
- the way these stamps are handled in the taiko editor is by displaying the value for each object, which creates a massive amount of unnecessary clutter, here's a couple of examples:

![image](https://user-images.githubusercontent.com/62819481/188055915-e773bad0-930d-41b7-9fb3-20de668e296e.png)
![image](https://user-images.githubusercontent.com/62819481/188055948-be9a1314-b6a8-423f-93de-027ee9bcb758.png)

the best way to handle this is to show those stamps when a change occurs, not for every object in existence, here's an example:

![image](https://user-images.githubusercontent.com/62819481/188057459-ba2307eb-a4c1-49c9-8880-8a36f3e30445.png)

Handling it this way would make the most sense from a taiko mapping perspective.

**Comment by peppy** [comment]
The current display methods are very temporary (I made the ad-hoc designs myself just to allow us to move the actual data to per-hitobject). Grouping them as you propose is potentially a good way forward, but requires further thought, and probably a finalised design before attempting it.

Note that this doesn't just apply to taiko but all rulesets. The timeline is ruleset agnostic (and we hope to keep it as such).
